apps/demo/src/app/cart/components/cart-wrapper-item-count/cart-wrapper-item-count.component.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, Input, Directive, DebugElement, OnInit } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { StoreModule, combineReducers, MemoizedSelector, Store } from '@ngrx/store';
+
+import { DaffCartFactory } from '@daffodil/cart/testing';
+
+import { CartWrapperItemCountComponent } from './cart-wrapper-item-count.component';
+import { provideMockStore, MockStore } from '@ngrx/store/testing';
+import { cold } from 'jasmine-marbles';
+
+@Component({ template: '<demo-cart-wrapper-item-count [itemCount]="itemCountValue"></demo-cart-wrapper-item-count>' })
+class WrapperComponent implements OnInit {
+  @Input() itemCountValue: number;
+
+  ngOnInit () {
+    // is there a better way to set default input values?
+    this.itemCountValue = 0
```

Comment:
Is this how you are supposed to initialize default values for inputs? Setting the value in the class def did not work.

Reply:
Yea, so you should be able to do either:
```
@Component({ template: '<demo-cart-wrapper-item-count [itemCount]="itemCountValue"></demo-cart-wrapper-item-count>' })
class WrapperComponent {
  itemCountValue: number = 0;
}
```
or set it in the `beforeEach` before you call `fixture.detectChanges()`
```
@Component({ template: '<demo-cart-wrapper-item-count [itemCount]="itemCountValue"></demo-cart-wrapper-item-count>' })
class WrapperComponent {
  itemCountValue: number;
}
...
beforeEach(() => {
    fixture = TestBed.createComponent(WrapperComponent);
    wrapper = fixture.componentInstance;
    store = TestBed.get(Store);
    wrapper.itemCountValue = 0;

    cartWrapperItemCountElement = fixture.debugElement.query(By.css('demo-cart-wrapper-item-count'));
    cartWrapperItemCountComponent = cartWrapperItemCountElement.componentInstance;

    fixture.detectChanges();
  });
...
```